packages/@okta/vuepress-site/docs/reference/api/factor-profiles/index.md

```diff
@@ -393,6 +363,14 @@ Example:
     }
 ```
 
+##### Properties
+
+| Property | Type   | Description                             |
+| -------- | ------ | --------------------------------------- |
+| `source.simpleUserAttribute.name`   | String | The name of the user profile attribute which containe the email address to be used in the factor. |
```

Comment:
`contains`

I agree that this is probably the best way to handle this type of deep nesting.